src/Database/Driver.php

```diff
@@ -256,6 +263,16 @@ public function supportsSavePoints(): bool
         return true;
     }
 
+    /**
+     * Returns true if the server supports common table expressions.
+     *
+     * @return bool
+     */
+    public function supportsCommonTableExpressions(): bool
```

Comment:
Nitpick: The method and related property name is a bit too long. Can be shorten it to `supportsCTE()` or `supportsCte()`?

Reply:
If you insist :)

Reply:
Pretty please :slightly_smiling_face:

Reply:
done